target/linux/pistachio/config-5.15

```diff
@@ -63,25 +68,34 @@ CONFIG_DMA_OF=y
 CONFIG_DMA_VIRTUAL_CHANNELS=y
 CONFIG_DTC=y
 CONFIG_DWMAC_GENERIC=y
-CONFIG_EARLY_PRINTK=y
-CONFIG_EARLY_PRINTK_8250=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
+# CONFIG_FIT_IMAGE_FDT_BOSTON is not set
+# CONFIG_FIT_IMAGE_FDT_JAGUAR2 is not set
+# CONFIG_FIT_IMAGE_FDT_LUTON is not set
+# CONFIG_FIT_IMAGE_FDT_MARDUK is not set
```

Comment:
You have to activate `CONFIG_FIT_IMAGE_FDT_MARDUK` to build the dtb file. This should fix your build problem.

Reply:
Thanks! :)